kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue

```diff
@@ -4,19 +4,22 @@
     :immersivePage="true"
     immersivePageIcon="arrow_back"
     :immersivePageRoute="toolbarRoute"
+    :immersivePagePrimary="true"
```

Comment:
Are these changes supposed to be included here? They don't seem to be related to the task management.

Reply:
Yeah, I'm throwing these changes in to fix #6395. This makes the preview page of the Quiz-creation workflow match the preview page of the Lesson-creation workflow. It must have gone under the radar in testing.

Reply:
LGTM!

Reply:
Definitely appreciate the drive-by fixes.

meta note for the future:

Generally we should try to keep independent changes in independent PRs to make code reviews and git archaeology easier. However, I totally understand that it's usually quicker to throw multiple changes into a single omnibus PR, and it's crunch time.

One strategy I sometimes use is to make a bunch of commits related to multiple fixes on a single local branch, and then squash/re-arrange/cherry-pick them them into two or three separate branches as independent PRs. I personally use https://gitup.co/ which makes this kind of history rewriting super easy...